src/static-build-loader/loader.ts

```diff
@@ -79,14 +107,14 @@ export default function loader(this: LoaderContext, content: string, sourceMap?:
 	types.visit(ast, {
 		visitExpressionStatement(path) {
 			const { node, parentPath, name } = path;
-			let comment;
-			if (namedTypes.Literal.check(node.expression) && typeof node.expression.value === 'string') {
-				const hasPragma = HAS_PRAGMA.exec(node.expression.value);
+			const expressionValue = getExpressionValue(node);
+			if (expressionValue) {
+				const hasPragma = HAS_PRAGMA.exec(expressionValue);
 				if (hasPragma) {
 					const [, negate, flag] = hasPragma;
 					comment = ` ${negate}has('${flag}')`;
 					if (flag in features) {
-						elideNextImport = negate ? !features[flag] : features[flag];
+						elideNextImport = negate ? !!features[flag] : !features[flag];
```

Comment:
Inverts the logic of the has, so now it will only elide the import when the has expression evaluates to `false`